java/src-jni/test/TestH5P.java

```diff
@@ -0,0 +1,1408 @@
+            assertTrue("layout: " + layout_type, layout_type == HDF5Constants.H5D_CHUNKED);
+        }
+        catch (Throwable err) {
+            err.printStackTrace();
+            fail("H5Pget_chunk: " + err);
+        }
+    }
+
+    @Test
+    public void testH5P_file_space_strategy()
+    {
+        long[] threshold  = {0};
+        boolean[] persist = {false};
+        int strategy      = 0;
+        try {
+            strategy = H5.H5Pget_file_space_strategy(fcpl_id, persist, threshold);
+            assertTrue("strategy(default): " + strategy,
+                       strategy == HDF5Constants.H5F_FSPACE_STRATEGY_FSM_AGGR);
+            assertTrue("persist(default): " + persist[0], persist[0] == false);
+            assertTrue("threshold(default): " + threshold[0], threshold[0] == 1);
+            H5.H5Pset_file_space_strategy(fcpl_id, HDF5Constants.H5F_FSPACE_STRATEGY_PAGE, true, 1);
+            strategy = H5.H5Pget_file_space_strategy(fcpl_id, persist, threshold);
+            assertTrue("strategy: " + strategy, strategy == HDF5Constants.H5F_FSPACE_STRATEGY_PAGE);
+            assertTrue("persist: " + persist[0], persist[0] == true);
+            assertTrue("threshold: " + threshold[0], threshold[0] == 1);
+        }
+        catch (Throwable err) {
+            err.printStackTrace();
+            fail("testH5P_file_space_strategy: " + err);
+        }
+    }
+
+    @Test
+    public void testH5P_file_space_page_size()
+    {
+        long page_size = 0;
+        try {
+            page_size = H5.H5Pget_file_space_page_size(fcpl_id);
+            assertTrue("page_size(default): " + page_size, page_size == 4096);
+            H5.H5Pset_file_space_page_size(fcpl_id, 512);
+            page_size = H5.H5Pget_file_space_page_size(fcpl_id);
+            assertTrue("page_size: " + page_size, page_size == 512);
+        }
+        catch (Throwable err) {
+            err.printStackTrace();
+            fail("testH5P_file_space_page_size: " + err);
+        }
+    }
+
+    @Test
+    public void testH5Pset_efile_prefix()
+    {
+        String prefix = "tmp";
+        try {
+            H5.H5Pset_efile_prefix(lapl_id, prefix);
+        }
+        catch (Throwable err) {
+            err.printStackTrace();
+            fail("H5Pset_efile_prefix: " + err);
+        }
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void testH5Pset_efile_prefix_null() throws Throwable
+    {
+        H5.H5Pset_efile_prefix(lapl_id, null);
+    }
+
+    @Test
+    public void testH5Pget_efile_prefix()
+    {
+        String prefix = "tmp";
+        String pre    = "";
+
+        try {
+            H5.H5Pset_efile_prefix(lapl_id, prefix);
+            pre = H5.H5Pget_efile_prefix(lapl_id);
+        }
+        catch (Throwable err) {
+            err.printStackTrace();
+            fail("H5Pget_efile_prefix: " + err);
+        }
+        assertTrue("The prefix: ", prefix.equals(pre));
+    }
+
+    @Ignore
```

Comment:
why is this disabled?  It looks like it is bound to fail because you don't create a chunked layout, unless I'm missing something

Reply:
Since it is jni test, and I did not change them, that has been disabled for awhile. Would need to find when that was done.